### DISCUSSION TITLE
Disable all mob ai!

### DISCUSSION THREAD

**Discussion by YTPSDanDie** [OmerBenGera/WildStacker#943] [Pending Feature Requests]
I love Wildstacker over other stacker plugins but the thing swaying me away is a missing feature!!
I have a server with different game modes and skyblock I want to disable the AI of the mobs.
Is it possible to add this feature so that you can set which AI to disable in a file or temporarily disable all?
e.g.#
" #Should the AI of ALL MOBS on the server be disabled?"
" #The parts of the AI that are disabled can be further customized in the "ai-settings"
"   disable-all-mob-ai: true/false" disable by default.

Thank you for reading!

**Comment by OmerBenGera** [comment]
Look for the `nerfed-entities` section in the config file
